BankingSystem/bank/NormalInterestCalculator.java

```diff
@@ -0,0 +1,21 @@
+package BankingSystem.bank;
+
+import java.math.BigDecimal;
+
+public class NormalInterestCalculator implements InterestCalculator{
+
+    @Override
+    public BigDecimal getInterest(BigDecimal balance) {
+        if(balance.compareTo(new BigDecimal(10000000))>=0)
```

Comment:
큰 숫자는 "_"로 구분해서 쓰면 가독성에 좋습니다.

```suggestion
        if(balance.compareTo(new BigDecimal(10_000_000))>=0)
```